boards/v2/qemu/qemu_x86/Kconfig.defconfig

```diff
@@ -46,6 +55,9 @@ if BOARD_QEMU_X86_64
 config BOARD
 	default "qemu_x86_64"
 
+config X86_64
+	default y
```

Comment:
(non-blocking) would be far more concise to have these in a `Kconfig` file

Reply:
done